webknossos/__generate_client.py

```diff
@@ -64,6 +64,18 @@ def add_api_prefix_for_non_data_paths(openapi_schema: Dict) -> None:
         paths[f"/api{path}"] = paths.pop(path)
 
 
+def remove_zarr_tagged_endpoints(openapi_schema: Dict) -> None:
+    """This removes all endpoints tagged with zarr-streaming, in-place."""
```

Comment:
```suggestion
    """This removes all endpoints tagged with zarr-streaming, in-place, to reduce the size of the generated client."""
```
(at least that’s how I understand it?)

Reply:
Sorry, automerge was too fast here, I'll include this in my next PR.

Reply:
It’s not super important anyway. Sorry, I had not noticed that automerge was already active here